src/style/talawa-theme.css

```
@@ -0,0 +1,781 @@
+  --leftDrawer-bg: var(--color-white);
+  --leftDrawer-fixedModule-bg: var(--color-white);
+  --leftDrawer-scrollbar-color: var(--color-black-8);
+  --leftDrawer-optionList-bg: var(--color-gray-50);
+  --LeftDrawer-org-profileContainer-bg: var(--color-blue-muted);
+  --leftDrawer-profileContainer-bg: var(--color-gray-50);
+  --leftDrawer-secondaryText-color: var(--color-gray-mid);
+  --leftDrawer-inactive-button-hover-bg: var(--color-gray-50);
+  --leftDrawer-collapse-active-button-bg: var(--color-blue-muted);
+  --leftDrawer-inactive-button-bg: transparent;
+  --leftDrawer-active-button-bg: var(--color-blue-500);
+  --leftDrawer-button-text-color: var(--color-gray-900);
```

Stylelint (17.6.0)

[error] Unexpected duplicate "--leftDrawer-button-text-color" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)